guide/spanish/miscellaneous/create-and-upload-and-link-animated-gif-image/index.md

```diff
@@ -8,4 +8,4 @@ localeTitle: Crea y sube y vincula imágenes gif animadas.
     *   Recortar, redimensionar y optimizar tantas veces como sea necesario.
 4.  Descargar animado GIF.
 5.  Subir imagen a [imgur.com](http://imgur.com) .
-6.  Enlace de la imagen usando `![image title](https://i.imgur.com/RqgGcf5.gif)` .
\ No newline at end of file
+6.  Enlace de la imagen usando `![image title](https://i.imgur.com/RqgGcf5.gif)` .
```

Comment:
Did you mean to change this URL?

Reply:
Else please take the file out of the PR

Reply:
@thecodingaviator, To save time, I implemented your request to revert to the previous link across all languages on github. 

If this change does not result in any noticeable change from the user side, I would rather not modify the affected PRs any further.